packages/react/src/callout/docs/overview.mdx

```diff
@@ -96,3 +96,19 @@ The feature variant of the callout can be used to draw even more emphasis to the
 	<Text as="p">Description of the callout.</Text>
 </Callout>
 ```
+
+## Title
+
+By default, `Callout` renders `CalloutTitle` as an `h2`. If you need to change the title’s semantic tag to an `h3`, for instance, you can use the `CalloutTitle` component in the `title` prop to explicitly set it. For information on maintaining proper heading levels refer to [Structuring headings and labels](/content/content-structure#structuring-headings-and-labels).
+
+```jsx
+import { CalloutTitle } from '@ag.ds-next/react/callout';
```

Comment:
What's the reasoning behind making this one just `jsx` and including the `import`?

Reply:
Same as what was already done on in PageAlert. The import breaks a jsx live, which is why it's not rendered. I'm not a huge fan

Reply:
Hmm ok, weird, I missed that. I don't think either should be just `jsx`. We do need to fix the `import { ... }` at the top of each page, though. Is this a compensation effort for that?

Reply:
Perhaps?